pypeit/pypmsgs.py

```diff
@@ -183,15 +195,12 @@ def error(self, msg, cls='PypeItError'):
         premsg = '\n'+self._start + self._white_RD + '[ERROR]   ::' + self._end + ' '
         self._print(premsg, msg)
 
-        # Close log file
+        # Close QA plots
         # TODO: This no longer "closes" the QA plots
-        self.close()
+        close_qa(self.pypeit_file, self.qa_path)
```

Comment:
Can we delete the TODO?  Also, is there any worry about `self._log` not being closed?

Reply:
I left the TODO because I didn't know what it does. But looking at it it appears to do something so I removed the comment